lib/galaxy/config/__init__.py

```diff
@@ -707,6 +707,7 @@ def _load_schema(self):
 
     def _override_tempdir(self, kwargs):
         if string_as_bool(kwargs.get("override_tempdir", "True")):
+            raise Exception("How did we get here ?")
```

Comment:
If we can't/shouldn't get here, the method should be removed (otherwise it's more dead code). It's test-related; introduced here: https://github.com/galaxyproject/galaxy/commit/f34bb5e1f66e47ae13396f8ab8d8336aa635a255 , then modified (see discussion) here: https://github.com/galaxyproject/galaxy/pull/9631#issuecomment-617008412

Reply:
Debugging leftover, thanks for the catch.